services/circleci/circleci.service.js

```diff
@@ -28,20 +27,34 @@ module.exports = class CircleCi extends BaseJsonService {
   }
 
   static render({ status }) {
-    if (['success', 'fixed'].includes(status)) {
-      return { message: 'passing', color: 'brightgreen' }
-    } else if (status === 'failed') {
-      return { message: 'failed', color: 'red' }
-    } else if (['no_tests', 'scheduled', 'not_run'].includes(status)) {
-      return { message: status.replace('_', ' '), color: 'yellow' }
-    } else {
-      return { message: status.replace('_', ' '), color: 'lightgrey' }
+    let color = 'lightgrey'
+    if (status === 'passing') {
+      color = 'brightgreen'
+    } else if (
+      ['failed', 'infrastructure fail', 'canceled', 'timed out'].includes(
+        status
+      )
+    ) {
+      color = 'red'
+    } else if (status === 'no tests') {
+      color = 'yellow'
```

Comment:
Same here. I've attempted to do something sensible-ish, but again if you're thinking "'canceled' should obviously be grey" or whatever, feel free to make suggestions and I'll change them. Once we've got this nailed I'll also add tests for the colour logic. There is relatively little consistency about this on other services. Prior art to maybe consider:
* https://github.com/amio/badgen-service/blob/master/libs/live-fns/circleci.js
* https://github.com/circleci/frontend/blob/c189f3546afe49b64c8ee86d92ff67ed9d2eda78/src-cljs/frontend/models/build.cljs#L174-L181
(although they are both looking at 'status' as opposed to 'outcome')